locale/en/pl.cfg

```diff
@@ -0,0 +1,26 @@
+[entity-name]
+classic-nixie-tube=Lampa Jarzeniowa
```

Comment:
Do you think "[Lampa cyfrowa](https://pl.wikipedia.org/wiki/Lampa_cyfrowa)" is a better translation? Apparently "Jarzeniowa" is "fluorescent lamp", which a Nixie tube is not.